### PR TITLE
Explicit use of github-api-emojis/lib/map.json and added class to img

### PR DESCRIPTION
**Pull request by webcome**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 "use strict";
 
 const emojer = require("emojer")
-    , emojis = require("github-api-emojis/lib/map")
+    , emojis = require("github-api-emojis/lib/map.json")
```
Comment (IonicaBizau) [inline lib/index.js]: Why is this needed?

File: lib/index.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 module.exports = function showdownEmoji () {
     return [
         {
             type: "output"
-          , filter: text => emojer(text, "<img src=\"https://github.global.ssl.fastly.net/images/icons/emoji/__EMOJI_NAME__.png?v5\" alt=\":__EMOJI_NAME__:\" title=\":__EMOJI_NAME__:\" class=\"emoji-img emoji\">", emojis)
+          , filter: text => emojer(text, "<img class=\"emoji\" src=\"https://github.global.ssl.fastly.net/images/icons/emoji/__EMOJI_NAME__.png?v5\" alt=\":__EMOJI_NAME__:\" title=\":__EMOJI_NAME__:\" class=\"emoji-img emoji\">", emojis)
```
Comment (IonicaBizau) [inline lib/index.js]: The `emoji` class is already there, along with `emoji-img`.

File: lib/index.js (hunk 1 above)
Comment (webcome) [reply lib/index.js]: We use webpack-encore. When run the bellow command, the system doesn't find the dependency.

# ./node_modules/.bin/encore dev
Running webpack ...

This dependency was not found:

* github-api-emojis/lib/map in ./node_modules/showdown-emoji/lib/index.js

To install it, you can run: npm install --save github-api-emojis/lib/map

Comment (IonicaBizau) [reply lib/index.js]: Umm, well, that seems to be more like a webpack-encore issue, then. I would suggest opening an issue in their repository as well.

If adding `.json`solves the issue here, I'll gladly merge the PR. 😁

**Comment by webcome** [comment]
I removed the duplicated class.
Thank you.

File: lib/index.js (hunk 1 above)
Comment (webcome) [reply lib/index.js]: Thank you.

File: lib/index.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-module.exports = function showdownEmoji () {
-    return [
-        {
-            type: "output"
-          , filter: text => emojer(text, "<img src=\"https://github.global.ssl.fastly.net/images/icons/emoji/__EMOJI_NAME__.png?v5\" alt=\":__EMOJI_NAME__:\" title=\":__EMOJI_NAME__:\" class=\"emoji-img emoji\">", emojis)
+module.exports = function showdownEmoji() {
+    return [{
+        type: "output",
+        filter: function filter(text) {
```
Comment (IonicaBizau) [inline lib/index.js]: Umm, is there a specific reason for using ES5?

Comment (webcome) [reply lib/index.js]: I'm sorry. Webpack3 can't process uglifyJS with ES6. And I don't know how to transpile that code with babel.
